test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -299,162 +298,362 @@ foo``u{2195}abc
+        $result.Count | Should -Be 2
+        $result[0] | Should -Be 5
+        $result[1] | Should -Be 6
     }
 
-	It "Test a simple multiple assignment. (line 773)" {
-        $result = ExecuteCommand '$one,$two = 1,2,3; "One = $one"; "Two = $two"'
-		$result.Count | Should -Be 2
-		$result[0] | Should -Be "One = 1"
-		$result[1] | Should -Be "Two = 2 3"
+    It "Test a simple multiple assignment. (line 773)" {
+        $result = ExecuteCommand '$one, $two = 1, 2, 3; "One = $one"; "Two = $two"'
+        $result.Count | Should -Be 2
+        $result[0] | Should -Be "One = 1"
+        $result[1] | Should -Be "Two = 2 3"
     }
 
-	It "Tests script, global and local scopes from a function inside a script. (line 824)" {
-		"`$var = 'script';function func { `$var; `$var = 'local'; `$local:var; `$script:var; `$global:var };func;`$var;">$testfile
-		ExecuteCommand "`$var = 'global'"
-        $result = ExecuteCommand "$testfile"
-		$result.Count | Should -Be 5
-		$result[0] | Should -BeExactly "script"
-		$result[1] | Should -BeExactly "local"
-		$result[2] | Should -BeExactly "script"
-		$result[3] | Should -BeExactly "global"
-		$result[4] | Should -BeExactly "script"
+    It "Tests script, global and local scopes from a function inside a script. (line 824)" {
+        @'
+$var = "script"
+function func {
+    $var
+    $var = "local"
+    $local:var
+    $script:var
+    $global:var
+}
+func
+$var
+'@ | Set-Content -Path $testfile
+        ExecuteCommand '$var = "global"'
+        $result = ExecuteCommand $testfile
+        $result.Count | Should -Be 5
+        $result[0] | Should -BeExactly "script"
+        $result[1] | Should -BeExactly "local"
+        $result[2] | Should -BeExactly "script"
+        $result[3] | Should -BeExactly "global"
+        $result[4] | Should -BeExactly "script"
+    }
+
+    $TestCases = @(
+        @{
+            Command        = '
+                while ($true) {
+                    1
+
+                    while ($true) {
+                        2; break; 3
+                    }
+
+                    4; break; 5
+                }
+            '
+            ExpectedResult = "1", "2", "4"
+        }
+        @{
+            Command        = '
+                for (;;) {
+                    1
+
+                    for(;;) {
+                        2; break; 3
+                    }
+
+                    4; break; 5
+                }
+            '
+            ExpectedResult = "1", "2", "4"
+        }
+        @{
+            Command        = '
+                foreach ($a in 1..3) {
+                    1
+
+                    foreach ($b in 1..3) {
+                        2; break; 3
+                    }
+
+                    4; break; 5
+                }
+            '
+            ExpectedResult = "1", "2", "4"
+        }
+    )
+    It "Use break inside of a loop that is inside another loop. (line 945)" -TestCases $TestCases {
```

Comment:
We have the same name for some tests that is not good.
Also I see that text in $commands was changed that is not good for cleanup PR.

Reply:
The text was altered because the test cases were unreadable. I didn't change their functionality, though. 

The formatting is all that changed apart from removal of semicolons and in a few of them I replaced `while (1)` with `while ($true)` which are functionally identical. It may make more sense to put these in a separate PR, but I feel it's largely just cleanup so I'll leave that decision to you. 🙂